client/blocks/post-share/index.jsx

```diff
@@ -430,13 +445,15 @@ class PostShare extends Component {
 			hasFetchedConnections,
 			hasRepublicizeFeature,
 			hasRepublicizeSchedulingFeature,
+			siteSlug,
+			translate,
 		} = this.props;
 
 		if ( ! hasFetchedConnections ) {
 			return null;
 		}
 
-		const { siteSlug, translate, isJetpack } = this.props;
+		const { isJetpack } = this.props;
```

Comment:
because if `! hasFetchedConnections` is `TRUE` we would be avoiding declare a var unnecessarily.